JS/global-variables.js

```diff
@@ -0,0 +1,6 @@
+var results;
```

Comment:
use const instead of variable

Reply:
using of const will force me to initialize it with a value but the value of result will be set only after fetching of API so declaring it as var is the only option